src/defaults/defaults.jl

```diff
@@ -221,14 +221,15 @@ end
 ##########################################################################################
 
 function get_building_polys(buildings)
-    building_polys = fill(Point2f[], length(buildings));
-    
+    building_polys = GeometryBasics.Polygon[]
```

Comment:
I first thought that it would be nicer to keep the preallocation of this vector because we already know its length. However, I've tried this out and it seems as if it's not more performant at all but actually slower by a factor of ~2.5. I'm a bit confused by that but anyways, happy to see that pushing values into this vector works just fine because in the end it's easier to read. :)

Reply:
We cannot preallocate the memory because beforehand we do not know how many nodes each polygon has.